packages/story-editor/src/components/form/stories/tags.js

```diff
@@ -48,7 +48,7 @@ export const _default = () => {
         <Tags.Input
           id="tags-input"
           aria-describedby="tags-description"
-          name="story-tags"
+          name="web_story_tag"
         />
         <Tags.Description id="tags-description">
           {'Separate with commas or the Enter key.'}
```

Comment:
Should this line be translated?

Reply:
I was like "yes absolutely 💯 " but then realized that this is just a storybook story component. So no need.